client/payment-details/payment-method/index.js

```diff
@@ -12,14 +12,19 @@ import { Card, CardBody, CardHeader } from '@wordpress/components';
 import Loadable from 'components/loadable';
 import CardDetails from './card';
 import SepaDetails from './sepa';
+import GiropayDetails from './giropay';
 
 const PaymentDetailsPaymentMethod = ( { charge = {}, isLoading } ) => {
-	let PaymentMethodDetails = SepaDetails;
```

Comment:
Looks like before it was defaulting to `SepaDetails` when `charge.payment_method_details` or when `charge.payment_method_details.type` were falsy, now it's defaulting to `CardDetails` - is this safe?

Reply:
So I was able to confirm that all three payment methods still worked with this change. The choice for default was arbitrary, but felt that given the history of card being the primary payment method before this, having it be the default made sense here? If there are concerns though it's no issue to change it back. @superdav42 would there be any concerns with swapping the default to card here?

Reply:
> The choice for default was arbitrary, but felt that given the history of card being the primary payment method before this, having it be the default made sense here?

Got it, that makes sense, thanks! 👍

Reply:
Yea, no problems with changing the default to CardDetails.
I think a better pattern to use in this case is actually to not have a default but throw an exception that could be used to show a generic error when an unexpected type is encountered. I'm not sure if we have such exception handling in place but it'd be a good future enhancement. Merchants seeing a white page when an error occurs is never a good UX.

Reply:
In that case [`componentDidCatch`](https://reactjs.org/docs/error-boundaries.html#introducing-error-boundaries) would probably be a good solution